stable/akv2k8s/templates/controller-deployment.yaml

```diff
@@ -84,6 +84,8 @@ spec:
           - "--v={{ template "controller.logLevel" . }}"
           - "--logging-format={{ template "controller.logFormat" . }}"
           - "--watch-all-namespaces={{ .Values.watchAllNamespaces }}"
+          - "--kubeResyncPeriod={{ .Values.kubeResyncPeriod }}"
+          - "--azureKeyVaultResyncPeriod={{ .Values.azureKeyVaultResyncPeriod }}"
         ports:
```

Comment:
Argument name was wrong must be =>

          - "--kube-resync-period={{ $.Values. kubeResyncPeriod }}"
          - "--azure-resync-period={{ $.Values.azureKeyVaultResyncPeriod }}"

Reply:
Thank you, it's been changed!